index.html

```diff
@@ -128,16 +145,45 @@ <h1 class="work-section-title">My Recent<br>Work</h1>
                 <div class="btn-green"><a>See Project</a></div>
             </div>
         </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <ul class="card-lable-container">
+                    <li class="card-lable">Ruby on rails</li>
+                    <li class="card-lable">css</li>
+                    <li class="card-lable">JavaScipt</li>
+                    <li class="card-lable">html</li>
+                </ul>
+                <div class="btn-green"><a>See Project</a></div>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <ul class="card-lable-container">
+                    <li class="card-lable">Ruby on rails</li>
+                    <li class="card-lable">css</li>
+                    <li class="card-lable">JavaScipt</li>
+                    <li class="card-lable">html</li>
+                </ul>
```

Comment:
- [ ] [OPTIONAL] Please, consider fixing spaces between the technologies in cards in desktop version:
<img width="1099" alt="Screen Shot 2022-02-18 at 20 07 14" src="https://user-images.githubusercontent.com/49064106/154719988-36756a1a-e0dc-444a-abf9-86a6709e1990.png">